source/transactions-convenient-api/tests/transaction-options.yml

```diff
@@ -148,9 +148,9 @@ tests:
                   document: { _id: 1 }
                 result:
                   insertedId: 1
-        transactionOptions:
-          readConcern: { level: snapshot }
-          writeConcern: { w: 1 }
+          options:
```

Comment:
Indentation was definitely off here and needed correction, but I'm going to revert the rename before merging. This argument is definitely named `transactionOptions` in the `withTransaction()` spec, and I'd like to keep it that way as I expect we may need to add other options down the line (e.g. client-side operation timeouts).

Reply:
Reverted in https://github.com/mongodb/specifications/commit/a4a95b01c5dabb1edfdd7676408ce303193f1957.